lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1618,7 +1620,7 @@ LLVMToSPIRVBase::transValueWithoutDecoration(Value *V, SPIRVBasicBlock *BB,
     spv::LoopControlMask LoopControl = getLoopControl(Branch, Parameters);
 
     if (Branch->isUnconditional()) {
-      // For "for" and "while" loops llvm.loop metadata is attached to
+      // Usually for "for" and "while" loops llvm.loop metadata is attached to
       // an unconditional branch instruction.
       if (LoopControl != spv::LoopControlMaskNone) {
```

Comment:
It may also be worth adding a comment about "unusual" cases being handled in the below block.